kaldb/src/test/java/com/slack/kaldb/chunkManager/IndexingChunkManagerTest.java

```diff
@@ -1359,9 +1361,9 @@ public void testRollOverFailure()
     assertThatThrownBy(
             () -> {
               int newOffset = 1;
-              final List<LogMessage> newMessage =
-                  MessageUtil.makeMessagesWithTimeDifference(11, 12, 1000, startTime);
-              for (LogMessage m : newMessage) {
+              List<Trace.Span> newMessage =
+                  SpanUtil.makeSpansWithTimeDifference(11, 22, 1000, startTime);
```

Comment:
Why'd this second param change here?

Reply:
typo :) I guess it didn't matter though heh!